en/phinx/configuration.rst

```diff
@@ -211,6 +211,16 @@ network connections. The socket path is configured with ``unix_socket``:
             unix_socket: /var/run/mysql/mysql.sock
             charset: utf8
 
+.. note::
+ 
+    Use the following bash command in Linux environment to retrieve socket file path:
+ 
+.. code-block:: bash
+
+    find / -name .s.PGSQL.5432 -ls 2>&1 | grep -v "Permission denied" # For PostGreSQL
```

Comment:
Could the # For PostgreSQL and # For MySQL comment be put on the line before the command so they're easier to spot when this potentially scrolls as along command?

Reply:
@othercorey  ... Yes, having it before helps the users who will just scan through the page.
@ravage84 also added your suggestion :)